README.md

```diff
@@ -106,12 +110,62 @@ This term will reduce to:
 
 A match syntax for machine numbers.
 We match the case 0 and the case where the number is greater
-than 0, p binds the value of the matching number - 1:
+than 0, `n-1` binds the value of the matching number - 1:
```

Comment:
Mention that n is the name of the matched variable.

We match ... greater than 0. If `n` is the matched variable, `n-1` binds the value..."